client/post-editor/media-modal/detail/detail-file-info.jsx

```diff
@@ -102,6 +102,10 @@ class EditorMediaModalDetailFileInfo extends React.Component {
 						<th>{ this.props.translate( 'File Type' ) }</th>
 						<td>{ this.getItemValue( 'extension' ) }</td>
 					</tr>
+					<tr>
+						<th>{ this.props.translate( 'File Size' ) }</th>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **141** times:
`translate( 'File size:' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/49500)